docs/Project.toml

```diff
@@ -1,7 +1,11 @@
 [deps]
+AnalyzeRegistry = "e713c705-17e4-4cec-abe0-95bf5bf3e10c"
```

Comment:
This should _not_ be in this project.  The workflow to build the documentation is to activate this project, `dev` the package from the top-level directory and then instantiate this project.  Also, if `AnalyzeRegistry` is here, I get an error that the package isn't registered.

Reply:
Makes sense; it's in there from me following exactly that workflow locally 🙂. I didn't think it really hurt because if you `dev` before you `instantiate` there shouldn't be an issue (which the CI does)